{
    "error": {
        "authentication": {
            "loginToContinue": "Please login to continue using..."
        },
        "general": {
            "requiredDataMissing": "Required data seems to be missing."
        },
        "creating": {
            "task": "An error occurred while creating the task."
        },
        "exists": {},
        "generating": {
            "token": "An error occurred while generating the token."
        },
        "inUse": {
            "username": "This username is already in use."
        },
        "invalid": {
            "credentials": "Invalid credentials!"
        },
        "retrieving": {
            "task": {
                "single": "An error occurred while retrieving the task.",
                "multiple": "An error occurred while retrieving the tasks."
            },
            "userDetails": {
                "single": "An error occurred while retrieving the user details."
            }
        },
        "saving": {
            "task": "An error occurred while saving the task.",
            "user": "An error occurred while saving the user."
        },
        "unavailable": {
            "task": {
                "single": "No such task found!",
                "multiple": ""
            },
            "tokenSecret": "The token secret is unavailable. Please set the environment variable.",
            "user": "No such user exists!"
        }
    },

    "success": {
        "deleted": {
            "task": {
                "single": "The task was deleted successfully!",
                "multiple": "The tasks were deleted successfully!"
            }
        }
    }
}
